force-app/main/default/layouts/JobApplication__c-Job Application Layout.layout-meta.xml

```diff
@@ -53,6 +54,114 @@
     <miniLayout>
         <fields>JobPostingRef__c</fields>
     </miniLayout>
+    <platformActionList>
+        <actionListContext>Record</actionListContext>
+        <platformActionListItems>
+            <actionName>JobApplication__c.NewInterview</actionName>
```

Comment:
Please add this button to the flexi page also

Reply:
Disregarding due to it now being the only "default" for the button since finding the solution